package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "flooper",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
I just told you in #10 that `npm publish` takes care of updating version numbers, and then you go and do this and break my :heart: .

Reply:
We haven't setup an account for npm. This will be the last, pinky promise.

Reply:
I believe there is one already - https://www.npmjs.com/org/whitesmith - good luck finding the credentials. For some reason `qnorr` is published under my account...